web/src/compose_fade.js

```diff
@@ -1,181 +0,0 @@
-import $ from "jquery";
-import _ from "lodash";
-
-import {buddy_list} from "./buddy_list";
-import * as compose_fade_helper from "./compose_fade_helper";
-import * as compose_fade_users from "./compose_fade_users";
-import * as compose_state from "./compose_state";
-import * as message_lists from "./message_lists";
-import * as message_viewport from "./message_viewport";
-import * as people from "./people";
-import * as rows from "./rows";
-import * as util from "./util";
-
-let normal_display = false;
-
-export function set_focused_recipient(msg_type) {
-    if (msg_type === undefined) {
-        compose_fade_helper.clear_focused_recipient();
-    }
-
-    // Construct focused_recipient as a mocked up element which has all the
-    // fields of a message used by util.same_recipient()
-    const focused_recipient = {
-        type: msg_type,
-    };
-
-    if (focused_recipient.type === "stream") {
-        const stream_id = compose_state.stream_id();
-        focused_recipient.topic = compose_state.topic();
-        if (stream_id) {
-            focused_recipient.stream_id = stream_id;
-        }
-    } else {
-        // Normalize the recipient list so it matches the one used when
-        // adding the message (see message_helper.process_new_message()).
-        const reply_to = util.normalize_recipients(compose_state.private_message_recipient());
-        focused_recipient.reply_to = reply_to;
-        focused_recipient.to_user_ids = people.reply_to_to_user_ids_string(reply_to);
-    }
-
-    compose_fade_helper.set_focused_recipient(focused_recipient);
-}
```

Comment:
You're removing too much here -- `compose_fade` manages both the message feed fading and parts of the buddy list fading (`compose_fade_users`) -- we are only removing the latter feature.